frontend-site/src/modules/staticData.js

```diff
@@ -53,6 +54,12 @@ export const servicesOffered = Object.freeze([
     description: 'Need a break a stressful week of classes? Pick up a board game, located in the LUG office, and grab some friends. There are plenty of games to choose from.',
     url: '',
   },
+  {
+    name: 'Minecraft',
+    image: require("@/assets/minecrat.jpg"),
+    description: 'If you still play this iconic game, come join the LUG Minecraft server. You can even check the server status or which members are online.',
```

Comment:
We can't check server status yet, so I'm not sure if we should be mentioning it here until it's implemented.

This may also be a question for @Benbigmac or @bmiddha (or maybe even @Walden1995), but what would be the best way to let people know how to connect to the server? Should we display the URL here, or tell people to ask one of the officers about the URL?

Reply:
Would LUG members be the only people able to use the server? If thats the case, maybe the link could be present but you must be logged in as a LUG member for the request to be accepted. If you arent logged in youll get a notification saying "Please login to access server" or something like that.